src/main/java/frc/robot/commands/VisionAssistedShooter.java

```diff
@@ -0,0 +1,124 @@
+
+import frc.robot.config.Config;
+
+import frc.robot.subsystems.DriveBase;
+import frc.robot.subsystems.ShooterSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.nettables.VisionCtrlNetTable;
+
+/**
+ * VisionAssistedShooter command.
+ * NOTE: this command should be run after command TurnToOuterPortCommand
+ */
+public class VisionAssistedShooter extends CommandBase {
+  
+  //subsystem
+  private final ShooterSubsystem shooter;
+    
+  //todo: can be configured in config file as well
+  //todo: measure the radius for the shooting wheel
+  private final double SHOOTER_ANGLE_IN_DEGREES  = 60.0;
+  private final double TARGET_HEIGHT_IN_METERS = 2.49;
+  private final double SHOOTER_WHEEL_RADIUS_IN_CM = 10;
+  private final double HALF_OF_GRAVITY = 4.91;
+  private final double CONVERSION_NUMBER = 3000;
+
+  //values from Vision Network Table
```

Comment:
minor detail on comment style, put a space after //
See also javadoc standards for method documentation